policystorage/nomad.go

```diff
@@ -63,7 +63,7 @@ func canonicalize(from *api.ScalingPolicy, to *Policy) {
 	}
 
 	if to.Target.Name == "" {
-		to.Target.Name = "local-nomad"
+		to.Target.Name = "nomad-target"
```

Comment:
Same as before. Have this value in a single place and just reference it in other places.